src/nvidia_rag/utils/vdb/vdb_base.py

```
@@ -133,6 +133,33 @@ def get_metadata_schema(
         """Retrieve the metadata schema for the specified collection."""
         pass
 
+    # ----------------------------------------------------------------------------------------------
+    # Methods for document info management
+    def create_document_info_collection(
+        self,
+    ) -> None:
+        """Create a document info collection."""
+        pass
```

Ruff (0.14.5)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)